spec/models/author_spec.rb

```diff
@@ -15,4 +15,14 @@
       expect(author.popular_books.count).to eq 5
     end
   end
+
+  describe "#find" do
+    it "should find author buy slug name" do
+      author = create(:author, full_name: "Stephen King")
+
+      result = Author.find('stephen-king')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.